addons/website_profile/static/src/js/website_profile.js

```diff
@@ -116,6 +116,19 @@ publicWidget.registry.websiteProfileEditor = publicWidget.Widget.extend({
     },
 });
 
+publicWidget.registry.websiteProfileNextRankCard = publicWidget.Widget.extend({
+    selector: '.o_wprofile_progress_circle',
+
+    /**
+     * @override
+     */
+    start: function () {
+        this.$('g[data-bs-toggle="tooltip"]').tooltip({delay: 0});
```

Comment:
Can you try to instead make this tooltip using only bootstrap classes?

Apparently it's possible to pass the delay as data in the template:

https://getbootstrap.com/docs/4.0/components/tooltips/#options

This will allow to get rid of this JS override (and also we tend to avoid jQuery as much as possible now that OWL is merged).

Reply:
Moved `delay` to xml, but I don't think currently we can initialize toolitp without jQuery (I also did not obverse any example without jQuery).